Cargo.toml

```diff
@@ -12,9 +12,16 @@ homepage = "https://fpm.dev"
 [dependencies]
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-ftd = { version = "0.1.11", path = "../ftd" }
+ftd = { version = "0.1.12", path = "../ftd" }
 clap = "2.33.3"
 home = "0.5.3"
+reqwest = { version = "0.11.6"}
```

Comment:
We should create a lint or auto formatter that converts such lines into `reqwest = "0.11.6"`.